tv_apps/browser/js/toolbar.js

```diff
@@ -108,6 +108,8 @@ var Toolbar = {
         this.clickMenuButtonBlock.bind(this));
     this.historyBlock.addEventListener('mouseup',
         Awesomescreen.selectHistoryTab.bind(Awesomescreen));
+    this.pocketBlock.addEventListener('mouseup',
+      this.clickPocketListBlock);
```

Comment:
To prevent confusion about ```this``` in the clickPocketListBlock and to follow the same event handler coding style, I suggest to change this line to ```this.clickPocketListBlock.bind(this));```.